src/components/AboutPage.js

```diff
@@ -1,15 +1,106 @@
 import React from 'react';
+import '../styling/about.css';
+import { Button, Col, Container, Row } from 'react-bootstrap';
+import { LinkContainer } from 'react-router-bootstrap';
+{/* TODO 
+  - images
+  - style cleanup
+  - spacing cleanup
+*/}
 
 const AboutPage = () => (
-  <div class="container mt-5">
-    <div class="row">
-      <div class="col"></div>
-      <div class="col-8">
-        <h1>About us</h1>
-      </div>
-      <div class="col"></div>
-    </div>
-  </div>
+  <Container fluid="md">
+
+    <Row>
+      {/* TODO place image here (logo) */}
+      <Col className="col"></Col>
+      <Col className="col-8">
+        <h1>Build for Black Lives</h1>
+        <p>
+          This is a blurb about who we are and why we started this project and
+          about how it is more about connecting people. Our mission and values 
+          about supporting Black lives. We want allies to educate themselves and
+           use this platform as a way to support Black lives.
+          <br />
```

Comment:
i'm just being nitpicky now lol you don't have to change this but usually it's better to use css and adjust margins/padding rather than adding linebreaks/br tags

Reply:
lol keep the suggestions coming. i'll adjust